code/src/components/Labyrinth.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { useSelector, useDispatch } from 'react-redux'
+
+import { fetchNext } from 'reducers/fetch'
+import { Button } from 'lib/Button'
+import { Card } from 'lib/Card'
+
+const Description = styled.h1`
+    font-size: 25px;
+`
+
+export const Labyrinth = () => {
+  const currentStep = useSelector((state) => state.game.currentStep)
+  const user = useSelector((state) => state.game.username)
+  const isLoading = useSelector((state) => state.ui.isLoading)
+  const dispatch = useDispatch()
+
+  const ColorChange = () => {
+    const element = document.body.style
+    if (currentStep.coordinates === '0,0') {
+      element.backgroundImage = 'linear-gradient(133deg, rgba(119,114,114,0.8981412394662921) 0%, rgba(112,108,108,0.8981412394662921) 0%, rgba(106,111,124,1) 23%, rgba(54,94,53,1) 71%, rgba(62,69,62,0.9831131495786517) 93%)'
```

Comment:
Maybe the value `'linear-gradient(133deg, rgba(119,114,114,0.8981412394662921) 0%, rgba(112,108,108,0.8981412394662921) 0%, rgba(106,111,124,1) 23%, rgba(54,94,53,1) 71%, rgba(62,69,62,0.9831131495786517) 93%)'` could be assigned to a constant?